variables.tf

```
@@ -42,3 +42,9 @@ variable "connector_timeout" {
   type        = string
   default     = "10m"
 }
+
+variable "max_throughput" {
+  description = "size of network throughput of the VPC Access Connector"
+  type        = number
+  default     = 1000
```

```suggestion
  default     = 300
```
Keep default as 300, which is the [default value from provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/vpc_access_connector#max_throughput).